### PR TITLE
Fix rubocop issues in spec files

### LINKED ISSUE #357: Fix rubocop issues in spec/**/*.rb

### PR DESCRIPTION
**Pull request by arun1595**
Fixes reasonable rubocop issues
Resolves #357

### CODE REVIEW

**Comment by KevinMulhern** [comment]
I'm not over the moon with the do end syntax around the let and subject blocks, I think it makes the tests a bit harder to read.

File: spec/controllers/confirmations_controller_spec.rb [hunk 1]
```diff
@@ -1,12 +1,9 @@
-require 'rails_helper'
-
-RSpec.describe ConfirmationsController do
-
+describe ConfirmationsController do
```
Comment (arun1595) [inline spec/controllers/confirmations_controller_spec.rb]: @KevinMulhern Sure. I'll update the PR with the suggested changes.

Comment (arun1595) [inline spec/controllers/confirmations_controller_spec.rb]: Done.

**Comment by KevinMulhern** [comment]
👍 LGTM

**Pull request merged**